src/rofi_rbw/clipboarder.py

```diff
@@ -77,10 +77,27 @@ def copy_to_clipboard(self, characters: str) -> None:
             encoding='utf-8'
         )
 
+        self.previously_copied = characters
```

Comment:
I'd like to declare this with the class definition, like
```
class XClipClipboarder(Clipboarder):
    __previously_copied: str
```

Also, I don't find the name ideal: At this moment, it's not "previously" copied, but currently. Maybe `last_copied_characters`? Or do you have a better idea?

Reply:
I think `last_copied_characters` is good, I've now changed it. Sorry I took a while to get back to this